docs/GettingStarted.md

```diff
@@ -171,76 +118,45 @@ or application. It responds to the Rack minimal interface, providing reasonable
 retrieved at the Rack level.
 To start using the middleware in your generic Rack application, add it to your ``config.ru``:
```

Comment:
We should say that this integration is automatically activated when using `Rails` (in this section or in the Rails section).